src/components/ec-summary-info/types.ts

```diff
@@ -0,0 +1,13 @@
+import type { IconName } from '../ec-icon/types';
```

Comment:
can we add a story for the sensitive items?

Reply:
I added controllers so we could test it in the storybook: 
![image](https://user-images.githubusercontent.com/99659688/224791649-c42a601f-a8ff-4a01-9976-b4dd267f2cf4.png)